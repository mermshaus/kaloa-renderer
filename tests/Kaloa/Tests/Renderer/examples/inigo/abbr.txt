[abbr="PHP>=5.4"]Traits[/abbr]

[abbr title="PHP>=5.4"]Traits[/abbr]

[abbr]Traits[/abbr]
