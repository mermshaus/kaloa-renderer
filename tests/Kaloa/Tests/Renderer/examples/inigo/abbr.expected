<p><abbr title="PHP&gt;=5.4">Traits</abbr></p>

<p><abbr title="PHP&gt;=5.4">Traits</abbr></p>

<p><abbr>Traits</abbr></p>